Flutter/lib/NameScreen.dart

```diff
@@ -1,30 +1,39 @@
-import 'package:flutter/material.dart';
-import 'package:google_fonts/google_fonts.dart';
-import 'CreateSettle.dart';
-import 'JoinSettle.dart';
-import 'app_localizations.dart';
-
-class NameScreen extends StatelessWidget {
-  final _formKey = GlobalKey<FormState>();
+import 'import_all.dart';
```

Comment:
Again, seems like bad practice to import everything

Reply:
Resolved in commit `app-style 89939db`